TROUBLESHOOTING.md

```diff
@@ -101,6 +101,16 @@ To specify your own Node.js executable, set this setting in your User settings.j
 "python.analysis.nodeExecutable": "<path to node.js exe>"
 ```
 
+For those using vscode-server remotely, you can increase the memory limit by setting the `NODE_OPTIONS` environment variable in your shell configuration.
+
+On Linux or Mac, add `export NODE_OPTIONS="--max-old-space-size=8192"` to either your `.xxx_profile` or `.xxxrc` file. On Windows, add `set NODE_OPTIONS=--max-old-space-size=8192` to your system environment variables.
```

Comment:
"add `set NODE_OPTIONS=--max-old-space-size=8192` to your system environment variables." Maybe remove `set` to avoid confusion?

Reply:
`set` is part of command. not part of english phrase.

Reply:
ok missed the first part that's in shell config.

Reply:
I am open to any other suggestion. should I say add `set ...` to your batch file to add it to your system environment variable?

Reply:
When I read the node.js link you put in there, my first thought was to open `System Properties` and copy the `Environment Variable` there, so yes I think what you have above is more clear to me.

Reply:
how about now?